LinkedList/LinkedList/TSLList.h

```diff
@@ -339,14 +320,33 @@ class TSLList
 		FNode* Temporal = Head;
 		for (int i = 0; i < Size; i++)
 		{
-			if (Predictate(Temporal))
+			if (Predicate(Temporal))
 			{
 				Remove(i);
 				i--;
 			}
 			Temporal = Temporal->Next;
 		}
-		return false;
 	}
 
+	private:
+		FNode* GetNode(const int Index)
+		{
+			FNode* Temporal = Head;
+			for (int i = 0; i < Index - 1; i++)
+			{
+				Temporal = Temporal->Next;
+			}
+			return Temporal;
+		}
+
```

Comment:
Good job on the GetNode private function! It is an easy way to avoid rewriting code, and you could inmediately change a lot of functions to use the iterator now by simply changing this method.